src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/CellUpdate.cs

```diff
@@ -86,6 +86,22 @@ public CellUpdate(DbColumnWrapper column, string valueAsString)
         #region Properties
 
         /// <summary>
+        /// Converts the cell update to a DbCellValue
+        /// </summary>
+        public DbCellValue AsDbCellValue
```

Comment:
the naming of this property suggests it should probably be a method (i.e. "As" is a verb)